testsuite/performance/tests/src/main/java/org/keycloak/performance/TestConfig.java

```diff
@@ -69,29 +70,25 @@
     public static final int refreshTokenCount = Integer.getInteger("refreshTokenCount", 0);
 
 
-    public static final String serverUris;
-    public static final List<String> serverUrisList;
+    public static final String servers;
+    public static final List<String> serverList;
 
     // Round-robin infinite iterator that directs each next session to the next server
-    public static final Iterator<String> serverUrisIterator;
+    public static final Iterator<String> serverIterator;
 
     static {
-        // if KEYCLOAK_SERVER_URIS env var is set, and system property serverUris is not set
-        String servers = System.getProperty("serverUris");
-        if (servers == null) {
-            String env = System.getenv("KEYCLOAK_SERVER_URIS");
-            serverUris = env != null ? env : "http://localhost:8080/auth";
+        // if KEYCLOAK_SERVER_URIS env var is set, and system property servers is not set
```

Comment:
Please revert all changes from serverUri(.*) to server(\1), these changes are irrelevant and the serverUri fits better the contents